src/ocean/core/ByteSwap.d

```diff
@@ -103,7 +104,7 @@ struct ByteSwap
 
     final static void swap16 (void *dst, size_t bytes)
     {
-        assert ((bytes & 0x01) is 0);
+        verify((bytes & 0x01) is 0);
```

Comment:
The commit message isn't really true. This commit alters more than just "in contracts".